01-introduction.Rmd

```diff
@@ -41,20 +56,30 @@ This book aims to go beyond teaching methods and code: by the end of it you shou
 Our approach differs from early adopters such as Stan Openshaw in one important way, however.
 At the turn of the 21^st^ Century it was unrealistic to expect readers to be able to reproduce code examples, due to barriers preventing access to the necessary hardware, software and data.
 Fast-forward two decades and things have progressed rapidly.
-Anyone with access to a laptop with ~4GB RAM can realistically expect to be able to install and run software for geocompuation on publicly accessible datasets, which are more widely available than ever before (as we'll see in Chapter \@ref(read-write)).^[
+Anyone with access to a laptop with ~4GB RAM can realistically expect to be able to install and run software for geocompuation on publicly accessible datasets, which are more widely available than ever before (as we will see in Chapter \@ref(read-write)).^[
 A laptop with 4GB running a modern operating system such as Ubuntu 16.04 onwards should also be able to reproduce the contents of this book.
 A laptop with this specification or above can be acquired second-hand for ~$100 in many countries nowadays, reducing the financial/hardware barrier to geocomputation far below the levels in operation in the early 2000s, when high-performance computers were unaffordable for most people.
 ]
 Unlike early works in the field all the work presented in this book is reproducible using code and example data supplied alongside the book, in R packages such as **spData**, the installation of which is covered in Chapter \@ref(spatial-class).
 
-Reproducible geographic research is related to Geographical Data Science (GDS).
-This recent concept essentially combines 'data science' with GIS and, like geocomputation, can be defined in comparison with GIS (see Table \@ref(tab:gdsl)).
-The focus on reproducibility and a command-line interface in this book is aligned with GDS.
-
-```{r gdsl, echo=FALSE, message=FALSE}
-d = readr::read_csv("extdata/gis-vs-gds-table.csv")
-knitr::kable(x = d, caption = "Differences in emphasis between the fields of Geographic Information Systems (GIS) and Geographic Data Science (GDS).")
-```
+One should be aware of the fact that Geocomputation has quite a few close cousins including Geographic Information Science (GIScience), Geomatics, Geoinformatics, Spatial Information Science, Geoinformation Engineering [@longley_geographic_2015], and of late also Geographic Data Science (GDS).
```

Comment:
`quite a few` is too informal. `many` would be better and more concise. Same with `of late`.

Minor comments I'm happy to 'retweak'.